docusaurus/docs/cloud/cli/cloud-cli.md

```diff
@@ -0,0 +1,76 @@
+---
+sidebar_label: 'Cloud CLI'
+displayed_sidebar: cloudSidebar
+sidebar_position: 3
+---
+
+# Command Line Interface (CLI)
+
+Strapi Cloud comes with a Command Line Interface (CLI) which allows you to log in and out, and to deploy a project that is not stored in a git repository. The CLI works with both the `yarn` and `npm` package managers.
```

Comment:
```suggestion
Strapi Cloud comes with a Command Line Interface (CLI) which allows you to log in and out, and to deploy a project that is not hosted on a remote git repository. The CLI works with both the `yarn` and `npm` package managers.
```

I would suggest this wording, because on your local machine the code is already stored in a git repository too 😅